introduction.md

```diff
@@ -1 +1,6 @@
 Hi, my name is Radek and my favorite CodeSchool path is JavaScript.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?